packages/imperative/src/config/cmd/init/init.definition.ts

```diff
@@ -46,20 +46,35 @@ export const initDefinition: ICommandDefinition = {
             description: "Prompt for secure values. Useful for disabling prompting in CI environments.",
             type: "boolean",
             defaultValue: true
+        },
+        {
+            name: "overwrite",
+            description: "Replace existing config files instead of merging the new changes.",
+            type: "boolean",
+            defaultValue: false,
+            conflictsWith: ["dry-run"]
```

Comment:
With overwrite commands, we often have a [confirmation option](https://github.com/zowe/imperative/blob/next/packages/imperative/src/config/cmd/auto-init/AutoInitConstants.ts#L81) to prevent people from invoking the option accidentally. In most places, this option is called "for-sure", with the alias "fs" for short. Would it make sense to add to this command as well? It could be potentially time consuming to create a configuration file again if overwrite is used.

Reply:
Good point, I added a for-sure flag to go with it. :)